voluptuous/validators.py

```diff
@@ -459,6 +459,35 @@ def PathExists(v):
         raise PathInvalid("Not a Path")
 
 
+class Maybe(object):
+    """Validate that the object is of a given type or is None.
+
+    :raises Invalid: if the value is not of the type declared or None
```

Comment:
This contradicts what is written above. None type should not raise an exception.